ts/src/program/namespace/account.ts

```diff
@@ -156,6 +157,20 @@ export class AccountClient<
     return data;
   }
 
+  /**
+   * Returns a deserialized program derived account.
+   *
+   * @param seeds The seeds of the program derived account to fetch.
+   */
+  async fetchPda(seeds: Array<Buffer | Uint8Array>, commitment?: Commitment): Promise<T> {
```

Comment:
Maybe this should return a two element array, including the PublicKey?

Reply:
That would be better - returned an array of `PublicKeys` for `fetchMultiplePda` as well. Too much to include the bump seeds too?

Reply:
Bump seed is probably unnecessary. There's no reason the client should ever need it.